lib/parse.js

```diff
@@ -68,7 +68,7 @@ module.exports = (input) => {
     device = _.chain(device)
       .split('\n')
       .filter((line) => {
-        return /^(\s\s-\s)?[a-z]+:/g.test(line);
+        return /^(\s\s-\s)?[a-zA-Z]+:/g.test(line);
```

Comment:
This can be changed to `/^(\s{2}-\s)?[a-z]/i` - no need for a global match flag when testing for the start of a string